packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -193,14 +193,17 @@ async function terraformAirnodeManage(
   await execTerraform(execOptions, command, prepareAirnodeManageArguments(cloudProvider, commonArguments));
 }
 
+type CloudProviderExtended = CloudProvider & {
+  readonly maxConcurrency?: number;
```

Comment:
Maybe add a comment that this is required for deploy but missing in removal - but it's a headache to type...